### ISSUE TITLE
Contributing to R Core redesign

### ISSUE THREAD

**Issue by PythonCoderUnicorn** [r-devel/rdevguide#25]
# Existing Resources Relevant to Contributing to R Core
-------------------------

## R (Core) Developer Page
A page for overall informal ideas and plans for the R statistical system and update news.
-  https://developer.r-project.org

## Contributing to R Core
A HTML book on how to use ``git`` on a Mac OS terminal for making commits, file directories, ``gitignore`` etc.
- **(svn)** subversion control system, contributing is done by ordinary patches from git commits and branches
- Contributing to GNU R: https://bookdown.org/lionel/contributing/

## Testing 
A Mirror of R svn server with Github actions CI for testing patches

### How it works
This mirror synchronises every 15 minutes with the official R svn server. Each new commit triggers a build on every platform, and the results are published on: https://r-devel.github.io, which shows an overview of the most recent revisions, including links to the build logs.

The Github icon in the last column links to the GitHub actions page for each build, where you can also download the generated Windows installer. These installers are not signed, they are only for testing.

- https://github.com/jeroen/r-svn

## Bug Reporting/ Review

1. Bug Reporting in R, a HTML page on what to do when you find a bug 
   - **TL;DR** if you see a ``.C`` or ``.Internal`` it's a bug for sure. 
   - **Document covers**: Making sure your problem is a bug, Where to submit bug reports (see spam protection)
   How to write useful and productive bug reports, How to submit patches, What to do if something goes wrong.
   - https://www.r-project.org/bugs.html
    

2. R can use your help reviewing bug reports
   - **TL;DR;** get bugzilla account, review older bugs and find ones outdated, find a bug with no examples but can be done
   
   - https://developer.r-project.org/Blog/public/2019/10/09/r-can-use-your-help-reviewing-bug-reports/index.html

3. Bugzilla mirror on GitHub
   If you have a bugzilla account this is a helpful resource for tracking R issues, look up issues by ``bugzilla_id`` or ``github_id``. 
   - https://github.com/MichaelChirico/r-bugs

**Comment by hturner** [comment]
This is an annotated version of our links page on the wiki (https://github.com/r-devel/rdevguide/wiki), a collection of refences that we though might be helpful in writing the guide.

@SaranjeetKaur this is worth reviewing when you review the guide as a whole: are all the links incorporated, or have they gone out of date so we can ignore them now?

I also like the the idea of TL;DR summaries - maybe we can add a callout box with the key points or a one-sentence summary at the start of each chapter?

**Comment by hturner** [comment]
I added a bit more commentary to the wiki page to explain what these links are about: https://github.com/r-devel/rdevguide/wiki. 

I think we have cited these references in the guide where that makes sense, though there may be some content from these resources that could still be incoporated in the guide. I think it is better to think about that when reviewing/revising particular chapters.

It's an open question whether TL;DR summaries at the start of each chapter would be useful. This may be addressed by switching the guide to Quarto as the in-chapter toc would give a quick overview of the contents . So I think we can leave this question for now and raise a new issue later as we see fit.
